src/components/Books.jsx

```diff
@@ -1,28 +1,16 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 
 const Books = () => {
-  const books = [
-    {
-      id: 1,
-      title: 'Romeo and Julliet',
-      author: 'William Shakespare',
-      category: 'Romance',
-    },
-    {
-      id: 2,
-      title: 'The Hunger Games',
-      author: 'Suzanne Collins',
-      category: 'Action',
-    },
-  ];
+  const { books } = useSelector((state) => state.book);
+  console.log(books);
```

Comment:
- [ ] I think it would be great if you remove the console.log from your code, debugging with the console is an excellent way, but after that, you should remove them to have clean and dry codebase 🔥🤩